app/src/main/java/de/hpi3d/gamepgrog/trap/datatypes/Language.java

```diff
@@ -0,0 +1,29 @@
+package de.hpi3d.gamepgrog.trap.datatypes;
+
+import org.parceler.Parcel;
+import org.parceler.ParcelConstructor;
+
+import java.util.Locale;
+
+@Parcel(Parcel.Serialization.BEAN)
+public class Language implements UserData {
+
+    private String languageCode;
+
+
+    public static Language getCurrentLanguage() {
+        String languageString = Locale.getDefault().getLanguage();
+        return new Language(languageString);
+    }
+
+    @ParcelConstructor
+    public Language(String languageCode) {
+        this.languageCode = languageCode;
+    }
+
+    @Override
+    public String[] requiredPermission() {
+        // No permission required.
+        return new String[0];
+    }
```

Comment:
I just noticed that this method is never used. I will remove it in Master after this Branch is merged